livekit-android-sdk/src/main/java/io/livekit/android/e2ee/E2EEManager.kt

```diff
@@ -96,9 +98,20 @@ constructor(keyProvider: KeyProvider) {
         }
     }
 
+    public fun removeSubscribedTrack(track: Track, publication: TrackPublication, participant: RemoteParticipant, room: Room) {
+        for (item in frameCryptors.entries) {
+            var key = item.key
+            var frameCryptor = item.value
+            if (key.first == participant.sid) {
```

Comment:
I think this should be `key.second`? trackId is in the first position and participantId in the second position.

Reply:
I think sid should be used here because it is unique and corresponds to the actual audio or video track. If the participant ID is used to remove the framecryptor, the video framecryptor may also be removed when the other party just unpublish the audio track, causing the video decoding to fail.

Reply:
I mean that you put `trackId to participantId` here:

https://github.com/livekit/client-sdk-android/pull/276/files#diff-6dbe0ea2e715879edad961dea930389cbd5bd274e22b5c7991d7649d1f6f88dfR172

This means `key.first` equals trackId, and `key.second` equals participantId. The if statement here would never pass since you're checking a track id against a participant id.

Reply:
maybe we just need trackId/sid as the frameCryptors Map key, or change to 
`if (key.first == participant.sid && key.second == publication.sid) {`

Reply:
I've fixed it and testing works fine